pkg/arvo/app/chat-view.hoon

```diff
@@ -109,7 +109,7 @@
       [[%give %fact ~ %json !>(*json)]~ this]
     (on-watch:def path)
     ::
-    ++  message-limit  25
+    ++  message-limit  50
```

Comment:
Is this because, on taller screens, this wouldn't provide enough messages to enable the scrollbar, and that somehow breaking the logic below? Seems like maybe we should be fixing that case instead of just throwing more messages at the problem, since this may not be enough for people with small font sizes, or vertical monitor setups or w/e.

Reply:
Not just on taller screens, on a regular macbook pro, if the messages are all sent by a single author and are all only one line, it doesn't provide enough messages. This is a reasonable fix for now, as it's an edge case.

Reply:
> This is a reasonable fix for now, as it's an edge case.

So we're negatively impacting performance across the board, to salvage the edge case. I see.

(I mean it's not gonna be as bad as the original `100` message-limit, but still...)